src/main/java/ai/deepcode/javaclient/core/DeepCodeUtilsBase.java

```diff
@@ -77,12 +71,10 @@ protected DeepCodeUtilsBase(
   public boolean isSupportedFileFormat(@NotNull Object file) {
     // DCLogger.getInstance().info("isSupportedFileFormat started for " + psiFile.getName());
     if (ignoreInfoHolder.isIgnoredFile(file) || isGitIgnoredExternalCheck(file)) return false;
-    final boolean result =
-        getFileLength(file) < MAX_FILE_SIZE
-            && (supportedExtensions.contains(getFileExtention(file))
-                || supportedConfigFiles.contains(pdUtils.getFileName(file)));
     // DCLogger.getInstance().info("isSupportedFileFormat ends for " + psiFile.getName());
-    return result;
+    return getFileLength(file) < MAX_FILE_SIZE
+      && (supportedExtensions.contains(getFileExtention(file)) || supportedConfigFiles.contains(pdUtils.getFileName(file)))
+      && pdUtils.getFileSize(file) > 0;
```

Comment:
I mean to create once:
`long fileLength = getFileLength(file)`
And then do `0 < fileLength && fileLength < MAX_FILE_SIZE`